src/Libraries/CoreNodeModels/Input/ColorPalette.cs

```diff
@@ -169,14 +169,5 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
 
             return new[] { AstFactory.BuildAssignment(GetAstIdentifierForOutputIndex(0), colorNode) };
         }
-
-        /// <summary>
-        ///     Indicates whether node is input node. 
-        ///     Used to bind visibility of UI for user selection.
-        /// </summary>
-        public override bool IsInputNode
```

Comment:
does this node actually execute heedlessly?

Reply:
as apposed to other input nodes?

Reply:
We use these in RDC and get tessellation data with color.  They are not input nodes but they run

Reply:
Why are these not input nodes? I'm just confused by this change.

Reply:
there is a color input type:
https://github.com/DynamoDS/Dynamo/pull/11398/files/c374680730481e808d9235df4ace55254a3c0d6b#diff-4ed48c5f49bfda18fef9e3525ec827e474feb30050cf2ce12dce35566bfb970aR20

Reply:
@mjkkirschner So the color type was originally designed and implemented in `NodeInputData` -> https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Graph/Nodes/NodeInputData.cs#L20.   This node also implemented an override to store that data -> https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodeModels/Input/ColorPalette.cs#L50.  The only change here is we set this node to not be able to set as an input node.  Not sure exactly why but I think it was because the player had not implemented a picker.  Since then the Revit player has added ability to show warnings for any nodes that are not supported.  So in practice the only thing we are changing is the ability of the user to set this node as an input..

Reply:
Right - but we don't know who else is using this behavior, and I don't think it's a bug. No particularly strong opinion on this case but changing behavior should be done carefully.

* What will happen to previous graphs that set this as an input?
* If some other extension or integration is using this behavior - what is the consequence for them?
* What is the benefit of this change?

Reply:
@saintentropy can you or @nate-peters answer these questions? I'm interested in what the benefit is and if it's worth any potential breaks because of the change in behavior.

Reply:
@nate-peters cleared this up.